cmd/stack_create_test.go

```diff
@@ -196,16 +221,26 @@ func TestStackCreateInvalidLongStackName(t *testing.T) {
 	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
 	defer cleanup()
 
-	args := []string{"stack", "create", "testing_stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stack"}
+	testStackName := "testing_stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stack"
+
+	args := []string{"stack", "create", testStackName}
 	_, err := cmdtest.RunAppsody(sandbox, args...)
 
 	if err == nil {
 		t.Fatal(err)
 	}
 
-	exists, err := cmdtest.Exists("testing_stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stack")
+	exists, err := cmdtest.Exists(testStackName)
+	if err != nil {
+		t.Fatal(err)
+	}
 
 	if exists {
+		// It SHOULDN'T exist, but it might
+		err = os.RemoveAll(testStackName)
+		if err != nil {
+			t.Fatal(err)
+		}
 		t.Fatal(err)
```

Comment:
A better message here would be nice as to why the test is failing. The `err` will be `nil` at this point.

Reply:
For these `TestStackCreate` tests, I was thinking I could remove them and replace them with table driven tests because they all attempt the same appsody command just with different parameters, rather than updating each of them to be clearer? I could open a new PR for it?

Reply:
So I have updated `stack_create_test` to have tabled tests but it relies on `sandbox.TestDataPath` in a couple of places which isn't in master - I'll either add them into this PR or wait until this is merged and then open a new one. It's up to you really, if you'd rather keep it on-scope it can wait?

Reply:
Probably wait for a new PR after this has been merged.

Reply:
Yep, I've got it waiting for this to go in :)